src/main/java/uk/gov/digital/ho/hocs/queue/migration/MigrationMessageHandler.java

```diff
@@ -14,17 +14,21 @@ public class MigrationMessageHandler extends BaseMessageHandler {
 
     private final MigrationCaseService migrationCaseService;
 
+    private final MigrationValidator migrationValidator;
```

Comment:
Nitpick: formatting here, could we have a new line after this?
```suggestion
    private final MigrationValidator migrationValidator;
    
```

Reply:
Added.